alerts/cluster-ingress-operator/IngressWithoutClassName.md

```diff
@@ -0,0 +1,33 @@
+# IngressWithoutClassName
+
+## Meaning
+
+This alert fires when there is an Ingress with an unset IngressClassName
+for longer than one day.
+
+## Impact
+
```

Comment:
Add:

If this is a valid Ingress resource, it needs to have an `ingressClassName` to stop the Alert. `ingressClassName` is the name of an `ingressClass` cluster resource.    Otherwise, delete the misconfigured Ingress.